File: Array.java
public class Array {
    public static int findLargest(int[] arr) {
        if (arr.length == 0) {
            System.out.println("Array is empty");
            return -1; // Return an error value
        }

        int maxVal = arr[0];
        for (int i = 1; i < arr.length; i++) {
            if (arr[i] > maxVal) {
                maxVal = arr[i];
            }
        }

        return maxVal;
    }

    public static void main(String[] args) {
        int[] arr = { 3, 1, 4, 1, 5, 9, 2 };
        int largest = findLargest(arr);
        System.out.println("Largest element: " + largest);
    }
}

File: Array.c
#include <stdio.h>
int find_largest(int arr[], int n) {
    if (n == 0) {
        printf("Array is empty\n");
        return -1; 
    }

    int max_val = arr[0];
    for (int i = 1; i < n; i++) {
        if (arr[i] > max_val) {
            max_val = arr[i];
        }
    }

    return max_val;
}

int main() {
    int arr[] = {3, 1, 4, 1, 5, 9, 2};
    int n = sizeof(arr) / sizeof(arr[0]);
    int largest = find_largest(arr, n);
    printf("Largest element: %d\n", largest);
    return 0;
}